libr/reg/reg.c

```diff
@@ -88,7 +88,7 @@ R_API void r_reg_item_free(RRegItem *item) {
 }
 
 R_API int r_reg_get_name_idx(const char *type) {
-	if (type)
+	if (type && *type && type[1] && !type[2])
```

Comment:
```suggestion
        r_return_val_if_fail (R_STR_ISNOTEMPTY (type), -1);
	if (type[1] && !type[2])
```
then I would add a `default` case in the below switch, instead of having a return at the end of function on its own.

Reply:
Great. This feature is also great. It's my first time to try this.